parastell.py

```diff
@@ -202,15 +204,80 @@ def native_export():
 
     dir = Path(export['dir'])
     filename = Path(export['h5m_filename'])
+    
+    def merge_layer_surfaces():
+        #the plasma will have IDs 1,2,3 and name in components dict 'plasma'
+        #the ends are 'planar surfaces'
+        #the swept surfaces are 'spline surfaces'
+        #when using cubit.get_surface_type(int)
+        
+        # get a tuple of volume ids, these will be in the import order
+        # and are assumed to increase in order from inside to outside
+        volumes = cubit.get_entities('volume')
+
+        # tracks the surface id of the outer surface of the previous layer
+        lastOuterSurface = None
+        
+        for name, vol in zip(components.keys(),volumes):
```

Comment:
For robustness, you may want to build a dictionary of `components.keys()` to `volumes` when you build them rather than always relying on the ordering.  Something could come along in the future that disrupts the ordering and it will be hard to find.

Reply:
found that this already occurs in the cubit_export function, changed code to make use of that